docs/user/site.rst

```diff
@@ -172,7 +173,13 @@ next_node \: package
     omitted, there will be no IPv4 or IPv6 anycast address. The MAC address
     defaults to ``16:41:95:40:f7:dc``; this value usually doesn't need to be
     changed, but it can be adjusted to match existing deployments that use a
-    different value.
+    different value. Each entry in the ``name``-field will be resolved to the
+    ipv4 and ipv6-address.
+    For this to work, clients must use the next-node as their resolver. In
+    batman-based networks this requires setting the central dhcp server to
+    deliver this address as DNS-server via DHCP-option. When running a radvd
+    inside the network, this should be set to deliver the next-node
+    ipv6-address via rdnss.
```

Comment:
There are still a few issues in the above text:
* IPv4, IPv6, DHCP should be capitalized consistently
* Many incorrect colons, should be "IPv6 address", "DNS server", "DHCP option"

Reply:
adjusted the capitalisation. for colons I need some help... I always put them in english text like salt on french fries....